nistats/utils.py

```diff
@@ -23,15 +23,39 @@ def _check_list_length_match(list_1, list_2, var_name_1, var_name_2):
             % (str(var_name_1), len(list_1), str(var_name_2), len(list_2)))
 
 
+def _read_events_table(table):
```

Comment:
Isn't this featue used in many locations of the codebase ?

Reply:
Didn't understand your suggestion.

I created this today because we now have two types of table-like files:
1. csv with index_col=0. (Many tests are also using such data)
2. tsv with index_col=False (BIDS compliant events files)

Are you saying I should unprotect it (remove the undescore) and make it a regular, reusable function?

Reply:
unprotect the function -> Only if it indeed used somewhere else.
 (I was just wondering)